portfolio/src/main/webapp/script.js

```diff
@@ -59,19 +59,39 @@ function showComments() {
                 body: document.getElementById("textfield").value});
 
     // Fetches server comment data and writes it to the page.
-    fetch(request).then(comments => comments.json()).then(jsoncomments => {
-        document.getElementById("thecomments").innerHTML += "<p>" + 
-        jsoncomments[jsoncomments.length - 1] + "</p";
-    })
+    fetch(request).then(res => {
+        console.log(res);
+        loadComments();
+    });
 }
 
 /**
  * Accesses the comments from the datastore and displays them.
  */
 function loadComments() {
     fetch("/data").then(dataHashMap => dataHashMap.json()).then(dataHashMapJson => {
-        for (comment of dataHashMapJson.Comments) {
-            document.getElementById("thecomments").innerHTML += "<p>" + comment + "</p";
+        console.log(dataHashMapJson);
+        const len = dataHashMapJson.Comments.length;
+        var comments = dataHashMapJson.Comments;
+        let commNum = Math.min(document.getElementById("CommentNumber").options[0].selected ? 5 : 10, len);
+        var commentString = "";
+        //console.log(commNum);
```

Comment:
Remove comment?

Reply:
Ah. I meant to remove those before pushing. Got ahead of myself.